src/shadowbox/server/main.ts

```diff
@@ -186,9 +186,12 @@ async function main() {
   await accessKeyRepository.start(new RealClock());
 }
 
+function relativePathFromOutlineRoot(file:string): string {
+  return path.join(OUTLINE_DIR, file);
+}
+
 function getPersistentFilename(file: string): string {
-  const stateDir = process.env.SB_STATE_DIR || DEFAULT_STATE_DIR;
-  return path.join(stateDir, file);
+  return path.join(relativePathFromOutlineRoot('persisted-state'), file);
```

Comment:
This is breaking backward compatibility. Old servers will have SB_STATE_DIR set, but not SB_OUTLINE_DIR. 

You need something like
```ts
const persistedDir =  process.env.SB_STATE_DIR || relativePathFromOutlineRoot('persisted-state')
return path.join(persistedDir, file);
```